pkg/core/core.go

```diff
@@ -92,3 +92,28 @@ func StackReset(d drive.DriveIntf, comID ComID) error {
 	}
 	return nil
 }
+
+func FindComID(d drive.DriveIntf, d0 *Level0Discovery) (ComID, ProtocolLevel, error) {
```

Comment:
Now that this function is exported, it should get a code comment stating its functionality.